app/views/admin/reports/_all_events.html.haml

```diff
@@ -27,7 +27,13 @@
           %td
             = link_to event.title, edit_admin_conference_program_event_path(@conference.short_title, event)
             %br
-            .small (Presented by #{event.speaker_names})
+            .small
+              (Presented by
```

Comment:
https://de.wikipedia.org/wiki/Don%E2%80%99t_repeat_yourself

As you use this three times, maybe you should move this code into a helper or partial?
Also please have a look into ``join``

``
["one", "two"].join(", ")
-> "one, two"
``